src/app/services/usergames/useEditUserGame.ts

```diff
@@ -0,0 +1,95 @@
+import { CustomAxiosResponse, ErrorResponse } from '@app/constants/global/types';
+import { userGameRoute } from '@app/constants/global/urls';
+import client from '@app/utils/authApi';
+import { useMutation } from '@tanstack/react-query';
+import { useCallback, useMemo } from 'react';
```

Comment:
Nitpick: we may want to import dependencies first and then hook and component.

Reply:
Fixed. I will look into the automatic sort so we dont have to worry about this later on when coding.